Equilobe.DailyReport.BL/Jira/JiraClient.cs

```diff
@@ -146,9 +146,51 @@ public List<JiraBasicIssue> GetWorklogsForUser(string projectKey, string author,
 
         public List<JiraIssue> GetWorklogsForMultipleUsers(string authors, string startDate)
         {
-            var request = GetIssuesWorklogByJql(JiraApiUrls.WorklogsForMultipleUsers(authors, startDate));
+            var result = new List<JiraIssue>();
 
-            return ResolveRequest<JiraResponse<JiraIssue>>(request).Issues;
+            var issues = GetIssuesKeysForMultipleUsers(authors, startDate);
+
+            foreach (var issue in issues)
+            {
+                var response = GetIssueWorklogs(issue.Key);
+
+                if (response != null && response.Worklogs.Any())
+                {
+                    var jiraIssueWorklog = new JiraIssue
+                    {
+                        Id = issue.Id,
+                        Key = issue.Key,
+                        Fields = new JiraFields
+                        {
+                            Worklog = response
+                        }
+                    };
+
+                    result.Add(jiraIssueWorklog);
+                }
+            }
+
+            return result;
+        }
+
+        private List<JiraIssue> GetIssuesKeysForMultipleUsers(string authors, string startDate)
+        {
+            var issues = new List<JiraIssue>();
+            var startAt = 0;
+            var hasMoreValues = true;
+
+            do
+            {
+                var request = GetIssuesIdsByJql(startAt, JiraApiUrls.WorklogsForMultipleUsers(authors, startDate));
+                var response = ResolveRequest<JiraResponse<JiraIssue>>(request);
```

Comment:
move this part to GetIssuesIdsByJql method, and return directly the resolved response (like how it is done in the other cases)